src/components/Auth/Login/Login.react.js

```diff
@@ -159,6 +160,26 @@ class Login extends Component {
     }
   };
 
+  callGetAdmin = accessToken => {
+    this.props.actions
```

Comment:
```
const { actions } = this.props;
actions.getAdmin(accessToken)
```
would look better.

Reply:
This is also fine, as the getAdmin is not called anywhere else in the function